include/onnxruntime/core/graph/constants.h

```diff
@@ -26,5 +26,8 @@ constexpr const char* kNGraphExecutionProvider = "NGRAPHExecutionProvider";
 constexpr const char* kNupharExecutionProvider = "NupharExecutionProvider";
 constexpr const char* kBrainSliceExecutionProvider = "BrainSliceExecutionProvider";
 constexpr const char* kTensorrtExecutionProvider = "TensorrtExecutionProvider";
+
+constexpr int kMaximumAiOnnxVersionSupported = 10;
```

Comment:
Why it is 10?

Reply:
Note this is branch `rel-1905`, which is a much older snapshot of ORT where 11 support was only partial (IIRC only a few ops out of them all), which was before the ONNX 1.6 specification was even published and finalized. So the little bit of 11 support contained in this branch is an uncertain state relative to the spec. We want to say cleanly that we support the published version of ONNX 1.4 (opset 10) in Vibranium.

Reply:
Now you need to find a way to make the tests pass :-(

Reply:
Well I found a way to make them pass (at least `onnxruntime_test_all.exe` runs locally without errors) by applying the same rules to the other `GetLatestOpsetVersions` overload, which is what the Model class indirectly used to determine its default opset when no specific opset was passed to the constructor (it was using 11 by default). Also updated to testdata .onnx files which were set to opset 11 unnecessarily (moving them back down to 10).

Note that in master, I since saw that Scott McKay added explicit versions in a number of places so that we continue to still get testing on older op versions, and so that may not apply quite the same there.

Reply:
Alas, I see that `onnxruntime_test_all.exe` doesn't run all the test_models, and there are a number of opset 11 models in there, which is too many to update O_o. I won't make it in time to pacify the tests. Considering abandoning this PR and keeping the logic exclusively in WinML for now.

`2019-11-14 14:24:55.967545903 [E:onnxruntime:Default, runner.cc:42 RunTestCase] Test raw failed:Load model from test_models/test_abs_uint32/raw/model.onnx failed:This is an invalid model. Error in Node: : No Op registered for Abs with domain_version of 11`

https://dev.azure.com/onnxruntime/onnxruntime/_build/results?buildId=96833